### DISCUSSION TITLE
Listener not working, M5Stick C plus

### DISCUSSION THREAD

**Discussion by bobletboy** [josephdadams/TallyArbiter#519] [Q&A] [answered]
I've got the TA server working on a Windows host. I've installed the M5Stick C plus listener client as per instructions, and it's connected via WiFi. At boot, though, al I get on the M5 screen is "This file from web exec" - and it doesn't appear as a "Connected listener" in TA. 

Can anyone point me in the right direction please? 
Thanks!

**Comment by josephdadams** [comment]
Which version of the TA server?

**Reply by ArieR1963** [reply]
I have the same issue with Tally Arbiter 3.0.4. with 2.0.16 the Listener is working with my M5Stick C

**Reply by josephdadams** [reply, marked as answer]
Keep in mind that there is an m5stick code that only works with TA 2, and another set of m5stick code that only works with TA 3. Get the right one

**Reply by ArieR1963** [reply]
Joseph thanks for your response, I received the M5stick this week. Where can I see if it is a TA 2 or 3?
But I am using a Macbook Pro with Ventura, that's why I had to use 3.0.4. I have now installed 2.0.16 on my old MacBook Pro and it works without any problems, but now need an extra computer for my stream. Do I need to make an adjustment somewhere for 3.0.4.?

**Comment by Jocke4f** [comment]
The protocol between the listener client and TA is not backwords compatible between v2 and v3 of TA.
